src/celeritas/phys/PhysicsData.hh

```diff
@@ -256,6 +256,13 @@ struct PhysicsParamsScalars
                    == static_cast<bool>(fixed_step_action));
     }
 
+    //! Lowest kinetic energy for e+/e- energy loss calculation: 1keV
+    using Energy = units::MevEnergy;
```

Comment:
Move the type alias to the top of the class (the comment is also being applied here accidentally)

Reply:
Actually maybe just skip the alias since it's only used one place (as opposed to being used internally with a bunch of `real_type`).

Reply:
Done.